### PR TITLE
Create wick3dr0se.json

### PR DESCRIPTION
**Pull request by wick3dr0se**

### CODE REVIEW

File: projects/wick3dr0se.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+	"$schema": "./project.schema.json",
```
Comment (ackledotdev) [inline projects/wick3dr0se.json]: Change to `../` please!

**Review by ackledotdev — DISMISSED** [review]
Change path to schema

File: projects/wick3dr0se.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+	"$schema": "./wick3dr0se.json",
```
Comment (ackledotdev) [inline projects/wick3dr0se.json]: `../project.schema.json`

**Review by ackledotdev — CHANGES_REQUESTED** [review]

**Review by wick3dr0se — COMMENTED** [review]
Thanks 😅

**Review by wick3dr0se — COMMENTED** [review]
Updated

**Review by ackledotdev — APPROVED** [review]

**Pull request merged**